bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -252,13 +252,13 @@ impl Room {
     }
 
     pub async fn member(&self, user_id: String) -> Result<RoomMember, ClientError> {
-        let user_id = UserId::parse(&*user_id).context("Invalid user id.")?;
+        let user_id = UserId::parse(&*user_id)?;
```

Comment:
Is there a point in removing these? Perhaps it would make sense to have a helper that always applies this context to the `UserId` parsing step instead of just removing this?